app/test_split_integer.py

```diff
@@ -2,20 +2,32 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+
+    number_of_parts = 5
+    value = 30
```

Comment:
Do not create additional variables in functions, pass values directly to the `split_integer`.